app/rest/openshift/src/main/java/io/syndesis/openshift/OpenShiftServiceImpl.java

```diff
@@ -266,6 +266,7 @@ private void ensureBuildConfig(String name, DeploymentData deploymentData, Strin
                 // TODO: This environment setup needs to be externalized into application.properties
                 // https://github.com/syndesisio/syndesis-rest/issues/682
                 .withEnv(new EnvVar("MAVEN_OPTS", config.getMavenOptions(), null))
+                .withEnv(new EnvVar("BUILD_LOGLEVEL", config.isDebug() ? "5" : "1", null))
```

Comment:
don't chain withEnv as every invocation overrides the previous one, use var-args instead, like:

```java
            .withEnv(
                new EnvVar("..."),
                new EnvVar("...")
```

Reply:
I didn't notice that second EnvVar
Sorry.

Reply:
Sorry!